### ISSUE TITLE
Error in example 03: aero & aero-structural optimization of BAR_USC not working

### ISSUE THREAD

**Issue by mayankchetan** [WISDEM/WISDEM#367]
# Description
The BAR_USC  example in folder `03_blade` for aero & aero-structural optimization throws an error via PreCompPy, 

> ERROR** lower-surf last sector node out of bounds   1.0001774904964817        1.0000000000000000

# Steps to reproduce issue
### Option 1, direct run:
1. Install `develop` branch of WISDEM following instructions in the readme.
2. run `python examples/03_blade/blade_driver.py`

### Option 2, Unit Tests:
1. Install `develop` branch of WISDEM following instructions in readme.
2. run `python test/test_all.py`

# Current behavior

### Option 1, direct run:
Aero & Aero-structural optimization stops with error: 
> ERROR** lower-surf last sector node out of bounds   1.0001774904964817        1.0000000000000000

### Option 2, Unit Tests:
Test stops at 21% progress.

> wisdem/test/test_drivetrainse/test_layout.py .........                                                                                                               [ 21%]
> wisdem/test/test_examples/test_examples.py 

This is similar to the tests failing via Github actions for WISDEM after replacing BAR0 with BAR_USC in commit 93dcc9b

Github actions unit test for the latest `develop` branch (commit 8a52e11) fails at the same test case but the overall test is flagged as "successful": https://github.com/WISDEM/WISDEM/runs/5696104478?check_suite_focus=true#step:8:39

# Expected behavior
1. No error while running aero & aero-structural optimization examples for BAR_USC.
2. Failure of any unit test is flagged as a failure of Github action.

# Code versions
- Python 3.8
- WISDEM 'deveop' branch, commit 8a52e11
- Ubuntu 20.20 & Windows 10

**Comment by gbarter** [comment]
Good thing you filed this Issue when you did otherwise you might be stuck having to resolve it. . .

**Comment by mayankchetan** [comment]
I came across a few more, might do a PR in a few weeks. 😅

**Comment by ptrbortolotti** [comment]
Mayank please let us know if we can close this issue. Thank you!

**Comment by mayankchetan** [comment]
Thank you, Pietro. The updated turbine works at my end.

We are seeing errors in newer installations of WISDEM, I believe it's due to some updates in the dependencies. I have tried solving this internally by forcing conda to install modules from an older working conda environment. This was my temporary solution to support the MDO course at UTD: https://github.com/mayankchetan/WISDEM/tree/moduleIssue4utdMDO

I'm going to close this since the primary issue has been resolved.
